examples/suites/k8s_example_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package suites
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+	"path/filepath"
+	"sigs.k8s.io/e2e-framework/pkg/conf"
+	"sigs.k8s.io/e2e-framework/pkg/env"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+	"testing"
+)
+
+func clientSet() *kubernetes.Clientset {
+	var kubeconfig *string
+	if home := homedir.HomeDir(); home != "" {
+		kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
+	} else {
+		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+	}
+	flag.Parse()
+
+	// use the current context in kubeconfig
+	config, err := clientcmd.BuildConfigFromFlags("", *kubeconfig)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	// create the clientset
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+	return clientset
+}
+// This test just runs a quick smoke test that pods are in kube-system, showing how to build tests on client-go
+func ClientGoExampleTest(t *testing.T) {
+	env := env.NewWithConfig(conf.New())
+	// TODO Add label example later once they are fully vetted
+	feat := features.New("ClusterIP service").
+		Assess("pod can connect to a service over clusterIP", func(ctx context.Context, t *testing.T) context.Context {
```

Comment:
I know this is just an example, but we should still try to make sure all the comments / descriptions are accurate. This assessment says "pod can connect to servier over clusterIP", but the actual assertion is just an API assertion for listing pods. Can we update this?

Reply:
ya i copied from other tests :) i can fix if yall want to merge this ?

Reply:
Which tests? I think usage of the framework is fine, but the description used doesn't reflect what is actually tested. Can we fix it here? I think examples will be important for how other people end up using this package. We should try to always keep the examples in good shape :)